jobs/build/release/Jenkinsfile

```diff
@@ -97,14 +91,18 @@ node {
             name = "${params.NAME}"
             from_release_tag = "${params.FROM_RELEASE_TAG}"
             description = "${params.DESCRIPTION}"
-            advisory = "${params.ADVISORY}"
+            advisory = params.ADVISORY? Integer.parseInt(params.ADVISORY.toString()) : 0
```

Comment:
Hi @vfreex just curious, why did you turn the advisory into an integer?

Reply:
that's part of the first commit, which is actually from a different PR that I just merged (however I think this may need a rebase). really only need to review the second commit :)

Reply:
This is only my personal preference. The advisory ID returned by `elliott get --json` is `int` and I just personally prefer static typing than dynamic typing.